app/view_objects/results_view.rb

```diff
@@ -135,6 +136,20 @@ def course_count
     courses.meta["count"]
   end
 
+  def site_distance(course)
+    query_location = Geokit::LatLng.new(latitude.to_f, longitude.to_f)
+
+    site_coordinates = course.sites do |site|
```

Comment:
Do we need this? based on the below suggestion...

Reply:
I agree, I don't think we need this.